README.md

```diff
@@ -9,9 +9,27 @@ expressed in YAML or JSON.
 
 ![](image.jpg)
 
-## Requirements
+## Requirements as shown by Composer:
 
-- PHP 7.1 or higher
+To check your server, simply run `composer check-platform-reqs`.
+
+Sample output:
+```
+> composer check-platform-reqs
```

Comment:
do we really need composer usage guide here?

Reply:
Good question. I just noticed that "Requirements" section only stated the PHP version, but there are more actually (extensions). Some of the extension requirements come from dependency packages. To be complete this section lacked some details. Thats why I thought maybe I could just run that command and see what are actual dependencies are.

I am not really sure it was a good idea. Do you suggest leaving only the PHP version there?

Reply:
Mentioning deps requirements has no profit, I think, since they can change requirements as they wan't while maintaining contracts. Same stays for required extension. For your own requirements I hope everyone is capable to look into `composer.json` and find out which PHP is supported. If they get to the `packagist.org` - all the requirements are listed there alongside the readme.